Dockerfile

```diff
@@ -158,13 +159,13 @@ RUN sed -i -r "/^#?compress/c\compress\ncopytruncate" /etc/logrotate.conf && \
 RUN curl -s https://letsencrypt.org/certs/lets-encrypt-x3-cross-signed.pem > /etc/ssl/certs/lets-encrypt-x3-cross-signed.pem
 
 COPY ./target/bin /usr/local/bin
+
 # Start-mailserver script
 COPY ./target/start-mailserver.sh ./target/docker-configomat/configomat.sh /usr/local/bin/
 RUN chmod +x /usr/local/bin/*
 
 EXPOSE 25 587 143 993 110 995 4190
 
-CMD /usr/local/bin/start-mailserver.sh
-
+CMD ["/bin/bash", "/usr/local/bin/start-mailserver.sh"]
```

Comment:
You test 123 is failing because this change.
You need to set the same command to the setup.sh https://github.com/tomav/docker-mailserver/blob/master/setup.sh#L10